SingularityUI/app/actions/ui/requestDetail.es6

```diff
@@ -8,15 +8,20 @@ import {
 import { FetchTaskCleanups, FetchScheduledTasksForRequest } from '../api/tasks';
 import { FetchRequestUtilization } from '../api/utilization';
 
-export const refresh = (requestId) => (dispatch, getState) => {
+export const refresh = (requestId, location) => (dispatch, getState) => {
```

Comment:
One option is to not pass in location and instead use default parameters like this
```
export const refresh = (requestId, taskHistoryPage = 1, taskHistoryCount = 10) =>
```
pageSize might also be a better name than count